lib/workers/pr/changelog/index.ts

```diff
@@ -1,3 +1,5 @@
+import URL from 'url';
+
```

Comment:
```suggestion
```
Use global url object. Module is deprecated.

https://nodejs.org/api/url.html#url_new_url_input_base